src/main/scala/pt/tecnico/dsi/openstack/nova/services/Quotas.scala

```diff
@@ -15,28 +15,66 @@ final class Quotas[F[_]: Sync: Client](baseUri: Uri, authToken: Header) extends
 
   private def buildUri(projectId: String, userId: Option[String]): Uri = (uri / projectId).withOptionQueryParam("user_id", userId)
 
+  // In a Rest API if the resource does not exist you return a BadRequest not a NotFound </sarcasm>
+  private def getOption[R: Decoder](uri: Uri): F[Option[R]] = {
+    import dsl._
+    import cats.syntax.applicative._
+    import cats.syntax.functor._
+    import org.http4s.Method.GET
+    import org.http4s.Status.{BadRequest, Successful}
+    import org.http4s.EntityDecoder
+
+    implicit val d: EntityDecoder[F, R] = unwrapped(Some(name))
+    GET(uri, authToken).flatMap(client.run(_).use {
+      case Successful(response) => response.as[R].map(Option.apply)
+      case BadRequest(_) => Option.empty[R].pure[F]
+    })
+  }
+
   /**
    * Shows quotas for a project.
    *
    * @param projectId UUID of the project.
    * @param userId id of user to list the quotas for.
    */
-  def get(projectId: String, userId: Option[String] = None): F[WithId[Quota]] =
+  def get(projectId: String, userId: Option[String] = None): F[Option[Quota]] =
+    getOption(buildUri(projectId, userId))
+  /**
+   * Shows quotas for a project assuming the project/user exist.
+   *
+   * @param projectId UUID of the project.
+   * @param userId id of user to list the quotas for.
+   */
+  def apply(projectId: String, userId: Option[String] = None): F[Quota] =
     super.get(wrappedAt = Some(name), buildUri(projectId, userId))
 
   /**
    * Shows quota usage for a project.
    * @param projectId UUID of the project.
    * @param userId id of user to list the quotas for.
    */
-  def getUsage(projectId: String, userId: Option[String] = None): F[WithId[QuotaUsage]] =
+  def getUsage(projectId: String, userId: Option[String] = None): F[Option[QuotaUsage]] =
```

Comment:
Codacy found an issue: [Do not prefix getters with 'get'](https://app.codacy.com/gh/ist-dsi/scala-nova-client/pullRequest?prid=5937397)